rlw.sh

```diff
@@ -216,6 +218,10 @@ main () {
 		else
 			main
 		fi;;
+		'Visit the GitHub page')
+			URL='https://github.com/alfonsojon/roblox-linux-wrapper'
+			[[ -x $BROWSER ]] && exec "$BROWSER" "$URL" #Find default browser
+			path=$(which xdg-open || which gnome-open) && exec "$path" "$URL"
```

Comment:
Just use `xdg-open` if it exists. It will automatically pass arguments to `gnome-open` (GNOME), `exo-open` (Xfce), etc. on respective desktop environments. I don't ever see `$BROWSER` defined, and either way I think that should be chosen only if `xdg-open` fails (which should be NEVER on a proper desktop installation).